.eslintrc.json

```diff
@@ -130,7 +130,8 @@
         "jsx-a11y/label-has-for": 0,
         "@typescript-eslint/no-unnecessary-condition": 0,
         "@typescript-eslint/no-unused-vars": 0,
-        "primer-react/no-deprecated-colors": ["error", {"skipImportCheck": true}]
+        "primer-react/no-deprecated-colors": ["error", {"skipImportCheck": true}],
+        "no-redeclare": 0
```

Comment:
Added this for code blocks so that we can write this block ⬇️ without adding a `// eslint-ignore` inside the code block

```js
import {ActionList} from '@primer/react/drafts'
// ESM
import {ActionList} from '@primer/react/lib-esm/drafts'
// or commonjs:
import {ActionList} from '@primer/react/lib/drafts'
```
```